docs/config/cli.md

```diff
@@ -14,26 +14,26 @@ Stencil's CLI is included in the compiler, and can be invoked with the `stencil`
 
 Builds a Stencil project. The flags below are the available options for the `build` command.
 
-| Flag | Description | Alias |
-|------|-------------|-------|
-| `--ci` | Run a build using recommended settings for a Continuous Integration (CI) environment. Defaults the number of workers to 4, allows for extra time if taking screenshots via the tests and modifies the console logs. | |
-| `--config` | Path to the `stencil.config.ts` file. This flag is not needed in most cases since Stencil will find the config. Additionally, a Stencil config is not required. | `-c` |
-| `--debug` | Adds additional runtime code to help debug, and sets the log level for more verbose output. | |
-| `--dev` | Runs a development build. | |
-| `--docs-readme` | Generate `readme.md` docs based on the component types, properties, methods, events, JSDocs, CSS Custom Properties, etc. | |
+| Flag | Description                                                                                                                                                                                                                                        | Alias |
```

Comment:
My editor did all this whitespace formatting w/o asking me 💢 

I can back this out if folks don't like it